chat-plugins/info.js

```diff
@@ -166,6 +167,10 @@ exports.commands = {
 	hostsearch: 'ipsearch',
 	ipsearch: function (target, room, user, connection, cmd) {
 		if (!target.trim()) return this.parse('/help ipsearch');
+		if (net.isIP(target.trim()) === 0 && (!target.includes('.') || !target.includes('*'))) {
```

Comment:
I'm not sure this test is right. A host doesn't need to include `*`.

Might be easier just to clarify the "No results found" message.